### PR TITLE
Tempo: Add links to nodes in Service Graph pointing to Prometheus metrics

### PR DESCRIPTION
**Pull request by aocenas**
Add links into the node context menu navigating to a specific Prometheus metric.

Screenshot from 2021-11-01 11-17-49

### CODE REVIEW

**Comment by aocenas** [comment]
@mapno let me know if the link names make sense or whether something else could be added.

File: public/app/plugins/datasource/tempo/datasource.ts [hunk 1]
```diff
@@ -288,14 +288,36 @@ function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: s
...
   );
 }
 
+function makePromLink(title: string, metric: string, datasourceUid: string) {
+  return {
+    url: '',
+    title,
+    internal: {
+      query: {
+        expr: metric,
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/tempo/datasource.ts:313]: Do we want to run here range or instant query or both, cause ideally we should clarify this here.

**Review by ivanahuckova — COMMENTED** [review]
Left 1 comment, otherwise code and functionality looks good!

**Comment by ivanahuckova** [comment]
Also - non-related to this PR, but when my prom queries were failing (I had incorrect url), the errors weren't shown in service graph, I just got `0` results. The error would be helpful cause I wasn't sure what is happening.

**Review by mapno — APPROVED** [review]
LGTM

File: public/app/plugins/datasource/tempo/datasource.ts (hunk 1 above)
Comment (ivanahuckova) [reply public/app/plugins/datasource/tempo/datasource.ts:313]: Fixing this in Prometheus in https://github.com/grafana/grafana/pull/41263

**Review by ivanahuckova — APPROVED** [review]
LGTM

**Pull request merged**